rosdep/python.yaml

```diff
@@ -7858,6 +7871,19 @@ python3-venv:
   fedora: [python3-libs]
   nixos: [python3]
   ubuntu: [python3-venv]
+python3-virtualserialports-pip:
+  debian:
+    pip:
+      packages: [PyVirtualSerialPorts]
+  fedora:
+    pip:
+      packages: [PyVirtualSerialPorts]
+  gentoo:
+    pip:
```

Comment:
Did we want to remove the gentoo entry when it's pip?

Reply:
@mabelzhang Why should the pip entry on gentoo be removed?

Reply:
I was reading this comment at the time https://github.com/ros/rosdistro/pull/29514#discussion_r636348684
It sounds like it's a general gentoo+pip thing, and not specific to that PR?

Reply:
Shall I remove gentoo? I personally just use ubuntu and don't really care about gentoo, but added it in since it was also included for other pip PRs here, e.g.: https://github.com/ros/rosdistro/pull/29426/files

Reply:
Yes please remove pip for gentoo: https://github.com/ros/rosdistro/pull/29514#discussion_r636348684

Reply:
Gentoo is removed

Reply:
Thank you for iterating!